lib/inferno/result_summarizer.rb

```diff
@@ -17,7 +17,11 @@ def summarize
     private
 
     def prioritized_result_strings
-      Entities::Result::RESULT_OPTIONS
+      if all_optional_results?
+        Entities::Result::RESULT_OPTIONS.slice(0, 3) + Entities::Result::RESULT_OPTIONS.slice(3, 8).reverse!.rotate!
```

Comment:
This is really confusing. I'd prefer just an if to check for this special case in `#summarize`.

Reply:
done